### ISSUE TITLE
Enabling Telegram on Windows Crashes Pidgin (again)

### ISSUE THREAD

**Issue by Reilsss** [majn/telegram-purple#416]
Hello, I'm not sure where to address this,
so I thougth I should write here.

The problem I have is Identical to Issue
#324 https://github.com/majn/telegram-purple/issues/324
#220 https://github.com/majn/telegram-purple/issues/220
And kinda
#222 https://github.com/majn/telegram-purple/issues/222

I luckly do know 'why' pidgin crashes. But unfroently I do not know where the Pidgin Logfiles is located to copy past the debug info.

I readed thougth all this 3 threads and tryed the alternative plugin dll's posted in #220. Saddly without success. So I'm unsure what I can try to do to get away from the problem.

The problem appears for me when I join a secret telegram group of my friend.
The group contains only posted images of the resoultion 1024*1280 (or the like). And from what I can see from telegram client. There like 50+ of thise images posted in that group.

I preusme rigth now that pidgin crashes on start up because it atempts to sync itself with the messages of that group. (Means it atempts to download this 50 and more large images and explodes (crashes) while atempting that).

I can avoid the problem, currently only by leacing that group in the telegram main client. (So pidgin is not told to sync with the messages of that very group.)

I do run the plugin Downloadable from here: http://eion.robbmob.com/telegram/
together with Pidgin Version 2.12.0.0
I saddly can't figure out if its a 32 or 64  bit version of pidgin.

Does anyone maybe have a advice, about what I should try. Besides obvious stay away from picture heavy groups until a new version is avalible?

Thanks anyone in advance.

**Comment by EionRobb** [comment]
https://developer.pidgin.im/wiki/GetABacktrace

The backtrace is really the only way to work out where the crash it being caused.  Also if you can get the version number of the plugin from Help->Plugin Information, that would really help

**Comment by BenWiederhake** [comment]
Hey @Reilsss, does the bug still happen with [the new windows installer](https://github.com/majn/telegram-purple/pull/446#issuecomment-379464670)?

**Comment by azharisan** [comment]
do you live???

**Comment by Reilsss** [comment]
@Azharisan, I'm unsure, I'm still struggling with the aftermath situation of my fathers accident (he recied braindamage saddly). As a result of this I lost focus about many online related things like this. I'm sorry for the enterly long wait till responce.

@BenWiederhake: I tryed out the new windows installer. Sofar I have not seen any crashes.
I also visited the image focused group that used to give me the crashes. And sofar everything seems to work as it is supposed to.

Therefore I belive that the problem I faced has been fixed. If I should find something I guess I can always submit a new issue?
